### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by tharnath08**

### CODE REVIEW

File: test/test_hello_freertos.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+int main(void)
+{
+    stdio_init_all();
+    sleep_ms(10000); // Give time for TTY to attach.
+    printf("Start tests\n");
+    UNITY_BEGIN();
+    RUN_TEST(test_blink_regular);
+    RUN_TEST(test_blink_skipped);
```
Comment (corey-buchanan) [inline test/test_hello_freertos.c:54]: Can you add in the line to run the test you added?

**Review by Cromie1 — APPROVED** [review]
Code looks great. It looks like it tests properly. I will approve this but in the future I would test all the chars of the alphabet just to ensure it works properly. I would also test toggling each character in the opposite direction.

File: test/test_hello_freertos.c (hunk 1 above)
Comment (corey-buchanan) [reply test/test_hello_freertos.c:54]: Nevermind, I added it when I fixed the other tests (the gpio hadn't been initialized properly)

**Review by Cromie1 — APPROVED** [review]
I think these changes are good and make sense.

File: test/test_hello_freertos.c (hunk 1 above)
Comment (tharnath08) [reply test/test_hello_freertos.c:54]: My bad. Thanks for fixing.

**Pull request merged**